Adapter_Revit_UI/AdapterActions/Push.cs

```diff
@@ -40,19 +41,44 @@ public partial class RevitUIAdapter
 
         public override List<object> Push(IEnumerable<object> objects, string tag = "", PushType pushType = PushType.AdapterDefault, ActionConfig actionConfig = null)
         {
-            bool success = true;
+            // Check the document
+            UIDocument uiDocument = this.UIDocument;
+            Document document = this.Document;
+            if (document == null)
+            {
+                BH.Engine.Reflection.Compute.RecordError("BHoM objects could not be removed because Revit Document is null.");
```

Comment:
See https://github.com/BHoM/Revit_Toolkit/pull/572/files#r387096881

Reply:
Changed to _"BHoM objects could not be removed because Revit Document is null (possibly there is no open documents in Revit)."_